storage/rocksdb/rdb_datadic.h

```diff
@@ -387,7 +387,11 @@ class Rdb_key_def
     SECONDARY_FORMAT_VERSION_INITIAL= 10,
     // This change the SK format to include unpack_info.
     SECONDARY_FORMAT_VERSION_UPDATE1= 11,
-    SECONDARY_FORMAT_VERSION_LATEST= SECONDARY_FORMAT_VERSION_UPDATE1,
```

Comment:
So, I think there is a possibility that this could break something that was already working. I ended up removing it because it's not clear to me when we'd be able to remove backwards compatibility code, if we don't know who uses it.

We should probably decide together what our stance is towards backwards compatibility.